packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -918,7 +918,7 @@ const ReleaseDetailsPage = () => {
     });
 
     if ('data' in response) {
-      push('/plugins/content-releases');
+      replace('/plugins/content-releases');
```

Comment:
ahh right so now there is no going back!
<img src="https://media2.giphy.com/media/on741mxYDa0EicsPB1/giphy.gif"/>